8Knot/pages/landing/sections/definitions_section.py

```diff
@@ -0,0 +1,192 @@
+                                        html.Div(
+                                            [
+                                                html.H3(
+                                                    "CHAOSS",
+                                                    className="section-title",
+                                                    style={"font-size": "1.7rem !important"},
+                                                ),
+                                                html.P(
+                                                    "Community Health Analytics for Open Source Software. A Linux Foundation project focused on creating analytics and metrics to help define community health and sustainability for open source projects.",
+                                                    className="feature-body",
+                                                    style={"font-size": "1.4rem !important"},
+                                                ),
+                                            ],
+                                            className="feature-title",
+                                        ),
+                                    ],
+                                    width=12,
+                                )
+                            ],
+                            className="feature-section mb-4",
+                        ),
+                        # Augur Definition
+                        dbc.Row(
+                            [
+                                dbc.Col(
+                                    [
+                                        html.Div(
+                                            [
+                                                html.H3(
+                                                    "Augur",
+                                                    className="section-title",
+                                                    style={"font-size": "1.7rem !important"},
+                                                ),
+                                                html.P(
+                                                    "The data collection and processing system that powers 8Knot. Augur gathers data from various sources including Git repositories, GitHub, GitLab, and other platforms to create comprehensive datasets for analysis.",
+                                                    className="feature-body",
+                                                    style={"font-size": "1.4rem !important"},
+                                                ),
+                                            ],
+                                            className="feature-title",
+                                        ),
+                                    ],
+                                    width=12,
+                                )
+                            ],
+                            className="feature-section mb-4",
+                        ),
```

Comment:
theres a lot of structural repetition here when it seems like we are just using it in like an FAQ format - can we refactor this into a function or something so that the parts that are shared between each section are in one place and we can just repeatedly call the same function like we do with the sidebar nav?